spynnaker/pyNN/models/utility_models/delays/delay_extension_machine_vertex.py

```diff
@@ -115,11 +106,9 @@ def sdram_required(self):
                parse_extra_provenance_items)
     def parse_extra_provenance_items(self, label, x, y, p, provenance_data):
         (n_received, n_processed, n_added, n_sent, n_overflows, n_delays,
-         n_tdma_behind, n_sat, n_bad_neuron, n_bad_keys, n_late_spikes,
+         _n_tdma_behind, n_sat, n_bad_neuron, n_bad_keys, n_late_spikes,
```

Comment:
why an underscore param name?

Reply:
Ah this had been set to 0 but not removed.  I have removed it now.